### ISSUE TITLE
Fix dependency for spec tests

### ISSUE THREAD

**Issue by paulodiovani** [mjbellantoni/formtastic-bootstrap#132]
Tests are failing and it seens to be a dependency error.

```
~/.rbenv/versions/2.2.2/bin/ruby -I~/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-core-3.1.7/lib:~/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-support-3.1.2/lib ~/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-core-3.1.7/exe/rspec spec/actions/button_action_spec.rb spec/actions/input_action_spec.rb spec/actions/link_action_spec.rb spec/builder/action_spec.rb spec/builder/input_spec.rb spec/builder/semantic_fields_for_spec.rb spec/helpers/actions_helper_spec.rb spec/helpers/inputs_helper_spec.rb spec/helpers/semantic_errors_helper_spec.rb spec/inputs/base/wrapping_spec.rb spec/inputs/boolean_input_spec.rb spec/inputs/check_boxes_input_spec.rb spec/inputs/color_input_spec.rb spec/inputs/date_picker_input_spec.rb spec/inputs/date_select_input_spec.rb spec/inputs/datetime_picker_input_spec.rb spec/inputs/datetime_select_input_spec.rb spec/inputs/deprecated_time_date_datetime_inputs_spec.rb spec/inputs/email_input_spec.rb spec/inputs/file_input_spec.rb spec/inputs/hidden_input_spec.rb spec/inputs/number_input_spec.rb spec/inputs/password_input_spec.rb spec/inputs/phone_input_spec.rb spec/inputs/radio_input_spec.rb spec/inputs/range_input_spec.rb spec/inputs/search_input_spec.rb spec/inputs/select_input_spec.rb spec/inputs/string_input_spec.rb spec/inputs/text_input_spec.rb spec/inputs/time_select_input_spec.rb spec/inputs/time_zone_input_spec.rb spec/inputs/url_input_spec.rb
~/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/activesupport-3.2.21/lib/active_support/values/time_zone.rb:270: warning: circular argument reference - now
~/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-rails-3.1.0/lib/rspec/rails/adapters.rb:23:in `require': cannot load such file -- test/unit/assertions (LoadError)
	from ~/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-rails-3.1.0/lib/rspec/rails/adapters.rb:23:in `rescue in <module:Rails>'
	from ~/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-rails-3.1.0/lib/rspec/rails/adapters.rb:18:in `<module:Rails>'
	from ~/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-rails-3.1.0/lib/rspec/rails/adapters.rb:6:in `<module:RSpec>'
	from ~/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-rails-3.1.0/lib/rspec/rails/adapters.rb:5:in `<top (required)>'
	from ~/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/ammeter-0.2.9/lib/ammeter/rspec/generator/example/generator_example_group.rb:3:in `require'
	from ~/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/ammeter-0.2.9/lib/ammeter/rspec/generator/example/generator_example_group.rb:3:in `<top (required)>'
	from ~/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/ammeter-0.2.9/lib/ammeter/rspec/generator/example.rb:2:in `require'
	from ~/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/ammeter-0.2.9/lib/ammeter/rspec/generator/example.rb:2:in `<top (required)>'
	from ~/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/ammeter-0.2.9/lib/ammeter/init.rb:1:in `require'
	from ~/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/ammeter-0.2.9/lib/ammeter/init.rb:1:in `<top (required)>'
	from ~/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/formtastic-3.1.3/spec/spec_helper.rb:20:in `require'
	from ~/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/formtastic-3.1.3/spec/spec_helper.rb:20:in `<top (required)>'
	from /mnt/userdata/diovani/Development/Git/formtastic-bootstrap/spec/spec_helper.rb:9:in `require'
	from /mnt/userdata/diovani/Development/Git/formtastic-bootstrap/spec/spec_helper.rb:9:in `<top (required)>'
	from /mnt/userdata/diovani/Development/Git/formtastic-bootstrap/spec/actions/button_action_spec.rb:2:in `require'
	from /mnt/userdata/diovani/Development/Git/formtastic-bootstrap/spec/actions/button_action_spec.rb:2:in `<top (required)>'
	from ~/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-core-3.1.7/lib/rspec/core/configuration.rb:1105:in `load'
	from ~/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-core-3.1.7/lib/rspec/core/configuration.rb:1105:in `block in load_spec_files'
	from ~/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-core-3.1.7/lib/rspec/core/configuration.rb:1105:in `each'
	from ~/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-core-3.1.7/lib/rspec/core/configuration.rb:1105:in `load_spec_files'
	from ~/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-core-3.1.7/lib/rspec/core/runner.rb:96:in `setup'
	from ~/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-core-3.1.7/lib/rspec/core/runner.rb:84:in `run'
	from ~/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-core-3.1.7/lib/rspec/core/runner.rb:69:in `run'
	from ~/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-core-3.1.7/lib/rspec/core/runner.rb:37:in `invoke'
	from ~/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-core-3.1.7/exe/rspec:4:in `<main>'
~/.rbenv/versions/2.2.2/bin/ruby -I~/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-core-3.1.7/lib:~/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-support-3.1.2/lib ~/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-core-3.1.7/exe/rspec spec/actions/button_action_spec.rb spec/actions/input_action_spec.rb spec/actions/link_action_spec.rb spec/builder/action_spec.rb spec/builder/input_spec.rb spec/builder/semantic_fields_for_spec.rb spec/helpers/actions_helper_spec.rb spec/helpers/inputs_helper_spec.rb spec/helpers/semantic_errors_helper_spec.rb spec/inputs/base/wrapping_spec.rb spec/inputs/boolean_input_spec.rb spec/inputs/check_boxes_input_spec.rb spec/inputs/color_input_spec.rb spec/inputs/date_picker_input_spec.rb spec/inputs/date_select_input_spec.rb spec/inputs/datetime_picker_input_spec.rb spec/inputs/datetime_select_input_spec.rb spec/inputs/deprecated_time_date_datetime_inputs_spec.rb spec/inputs/email_input_spec.rb spec/inputs/file_input_spec.rb spec/inputs/hidden_input_spec.rb spec/inputs/number_input_spec.rb spec/inputs/password_input_spec.rb spec/inputs/phone_input_spec.rb spec/inputs/radio_input_spec.rb spec/inputs/range_input_spec.rb spec/inputs/search_input_spec.rb spec/inputs/select_input_spec.rb spec/inputs/string_input_spec.rb spec/inputs/text_input_spec.rb spec/inputs/time_select_input_spec.rb spec/inputs/time_zone_input_spec.rb spec/inputs/url_input_spec.rb failed
```

It's impossible to submit features pull requests ultil this is fixed.

To reproduce:

```console
git clone https://github.com/mjbellantoni/formtastic-bootstrap.git
cd formtastic-bootstrap
bundle install
bundle exec rake spec
```

**Comment by sodabrew** [comment]
Seems to be `~/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-rails-3.1.0/lib/rspec/rails/adapters.rb:23:in `require': cannot load such file -- test/unit/assertions`

If you can submit a PR to resolve this issue I would be happy to merge it.

**Comment by paulodiovani** [comment]
Sure @sodabrew. That's my plan.
But I don't know how to fix this. :cry:

**Comment by sodabrew** [comment]
Resolved by #133
